app/views/users/_post.html.erb

```diff
@@ -0,0 +1,8 @@
+<%@specific_user.recent_post.each do |post|%>
+Post# <%=post.id%>
+<%= post.text%>
+Comments: <%= post.comments_counter%>
+Likes: <%= post.likes_counter%>
+<br>
+<%end %>
+<button>See all posts</button>
```

Comment:
- [ ] Same here, Clicking on `see all posts` button suppose to take you to the user's post index page(view) for this specific user, So please fix that.
[Hint] You can also use `link_to` here.